harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -241,20 +551,25 @@ def on_shutdown(callback_name: str, on_trial_shutdown: Callable) -> None:
             #
             # We create it before loading state because we don't want the training_iterator
             # shuffling values after we load state.
+
```

Comment:
delete this empty line between the comment and what it's talking about.

Also update the comment to talk about the training_enumerator as well.